### PR TITLE
only show template category cards where there is an actual template

### PR DESCRIPTION
**Pull request by bthaile**
filter down the template cards and only show ones where there is actually a template for.

image

image

image

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 🌈

 This link will always point to this PR's latest changes: https://peek.run/r9zh33m

File: packages/augur-ui/src/modules/create-market/components/form.tsx [hunk 1]
```diff
@@ -619,11 +619,12 @@ export default class Form extends React.Component<FormProps, FormState> {
               )}
               {mainContent === REVIEW && <Review />}
               {mainContent === TEMPLATE_PICKER && <TemplatePicker />}
-              {mainContent === SUB_CATEGORIES && <SubCategories />}
+              {mainContent === SUB_CATEGORIES && <SubCategories nextPage={this.nextPage}/>}
```
Comment (phoebemirman) [inline packages/augur-ui/src/modules/create-market/components/form.tsx:622]: its weird that sub category page goes to next page automatically but the market type page or template picker one doesn't

File: packages/augur-ui/src/modules/create-market/get-template.ts [hunk 2]
```diff
@@ -1,4 +1,19 @@
-import { CATEGORICAL } from 'modules/common/constants';
+import { CATEGORICAL, YES_NO } from 'modules/common/constants';
+import {
+  SOCCER,
+  AMERICAN_FOOTBALL,
+  SPORTS,
+  MARKET_TEMPLATES,
+  MARKET_SUB_TEMPLATES,
+  MARKET_TYPE_TEMPLATES,
+} from './constants';
```
Comment (phoebemirman) [inline packages/augur-ui/src/modules/create-market/get-template.ts]: can you use the full path. same in other spot

**Review by phoebemirman — CHANGES_REQUESTED** [review]

File: packages/augur-ui/src/modules/create-market/get-template.ts [hunk 3]
```diff
@@ -71,45 +87,92 @@ export interface TemplateInput {
...
   categoryTemplates: Template[],
   marketType
 ) => {
   const values = categoryTemplates.filter(t => t.marketType === marketType);
-  return values;
+  return values.map(v => ({ ...v, inputs: getInputsByType(v.inputsType) }));
 };
 
+const getInputsByType = (inputName: TemplateInputTypeNames) => {
+  return inputes[inputName];
```
Comment (phoebemirman) [inline packages/augur-ui/src/modules/create-market/get-template.ts]: this is spelled wrong

File: packages/augur-ui/src/modules/create-market/get-template.ts [hunk 4]
```diff
@@ -123,87 +186,172 @@ export const getTemplateReadableDescription = (template: Template) => {
...
+            ],
+          },
+        ],
+      },
     },
   },
 };
+
+const inputes = {
+  [TemplateInputTypeNames.TEAM_VS_TEAM_POINTS_BIN]: [
```
Comment (phoebemirman) [inline packages/augur-ui/src/modules/create-market/get-template.ts:264]: why are these types if they are only used once

Comment (bthaile) [reply packages/augur-ui/src/modules/create-market/get-template.ts:264]: I suspect they will be used multiple times.

File: packages/augur-ui/src/modules/create-market/components/form.tsx (hunk 1 above)
Comment (bthaile) [reply packages/augur-ui/src/modules/create-market/components/form.tsx:622]: I'd like the market type page to auto page as well. Template picker does auto-page

**Review by phoebemirman — APPROVED** [review]

**Pull request merged**